.github/workflows/ci-prover-reusable.yml

```diff
@@ -22,7 +50,7 @@ jobs:
       - name: Start services
         run: |
           docker-compose -f ${RUNNER_COMPOSE_FILE} pull
-          docker-compose -f ${RUNNER_COMPOSE_FILE} up --build -d geth zk postgres
+          docker-compose -f ${RUNNER_COMPOSE_FILE} up --build -d zk
```

Comment:
so we're not starting geth and postgres anymore?

Reply:
no, they're not needed for unit tests and lints